Simplenote/SPAppDelegate.m

```diff
@@ -462,7 +462,7 @@ - (void)setSelectedTag:(NSString *)selectedTag {
 
 - (void)bucket:(SPBucket *)bucket didChangeObjectForKey:(NSString *)key forChangeType:(SPBucketChangeType)change memberNames:(NSArray *)memberNames
 {
-    if ([bucket.name isEqualToString:NSStringFromClass([Note class])]) {
+    if ([bucket isEqual:[_simperium notesBucket]]) {
```

Comment:
Do you mean to use `self.simperium.notesBucket`? Because `self.notesBucket` would point to the app delegate and we don't have `notesBucket` property in the app delegate 🤔

Reply:
Right! I meant using `self.simperium` rather than accessing directly the ivar 😄